src/screens/MainScreen.js

```diff
@@ -408,13 +409,13 @@ class BalanceView extends React.Component {
       marginBottom: 24,
     },
     networkView: {
-      backgroundColor: 'rgba(227, 0, 82, 0.1)',
+      backgroundColor: 'rgba(140, 70, 255, 0.1)',
```

Comment:
Isn't it better to include it as a constant? It seems to be the same as in TokenSelect.js, isn't it?

Reply:
Created a method in constants to get this color depending on the alpha value.